src/main/java/http_server/Response.java

```diff
@@ -0,0 +1,97 @@
+package http_server;
+
+import java.time.LocalDateTime;
+import java.time.ZoneId;
+import java.time.ZonedDateTime;
+import java.time.format.DateTimeFormatter;
+import java.util.LinkedHashMap;
+import java.util.Map;
+
+public class Response {
+    RequestParser request;
+    Router router;
+    LinkedHashMap<String, String> headerCollection = new LinkedHashMap<>();
+    String status;
+    String responseBody = "";
+    String crlf = "\r\n";
+
+    public Response(RequestParser requestParser, Router router) {
+        System.err.println(router);
+        this.request = requestParser;
+        this.router = router;
+        this.status = "200 OK";
```

Comment:
What will you need to change in order to handle the huge variety of possible HTTP statuses?

Reply:
I know that this is a WIP and I imagine that you will refactor as you continue to build functionality, so I hope you don't take my comments as being too premature

Reply:
Yeah, I was going to continue to adjust it as I hit future stories. I haven't quite figured out handling the different statuses yet or where those handlers would live yet, but was going to implement more and see if there's a pattern that emerges